### PR TITLE
Fix BridgeSupportReleaseBtcTest

### PR DESCRIPTION
**Pull request by julia-zack**

### CODE REVIEW

File: rskj-core/src/test/java/co/rsk/peg/BridgeSupportReleaseBtcTest.java [hunk 1]
```diff
@@ -98,6 +99,8 @@ class BridgeSupportReleaseBtcTest {
     private BridgeSupportBuilder bridgeSupportBuilder;
     private SignatureCache signatureCache;
     private FeePerKbSupport feePerKbSupport;
+    private FederationSupportBuilder federationSupportBuilder;
+    private FederationSupport federationSupport;
```
Comment (marcos-iov) [inline rskj-core/src/test/java/co/rsk/peg/BridgeSupportReleaseBtcTest.java:103]: I'd remove this one, each test uses it's own

File: rskj-core/src/test/java/co/rsk/peg/BridgeSupportReleaseBtcTest.java [hunk 2]
```diff
@@ -1340,10 +1358,18 @@ private BridgeSupport initBridgeSupport(BridgeEventLogger eventLogger, Activatio
             .withEventLogger(eventLogger)
             .withActivations(activationMock)
             .withSignatureCache(signatureCache)
+            .withFederationSupport(federationSupport)
             .withFeePerKbSupport(feePerKbSupport)
             .build();
     }
 
+    private FederationSupport initFederationSupport() {
```
Comment (marcos-iov) [inline rskj-core/src/test/java/co/rsk/peg/BridgeSupportReleaseBtcTest.java:1366]: This is probably not needed either

Comment (julia-zack) [reply rskj-core/src/test/java/co/rsk/peg/BridgeSupportReleaseBtcTest.java:1366]: done

File: rskj-core/src/test/java/co/rsk/peg/BridgeSupportReleaseBtcTest.java (hunk 1 above)
Comment (julia-zack) [reply rskj-core/src/test/java/co/rsk/peg/BridgeSupportReleaseBtcTest.java:103]: done

**Review by marcos-iov — APPROVED** [review]

**Pull request merged**